client/sharkey_client.go

```diff
@@ -80,22 +80,21 @@ func main() {
 		log.Print("Generated http client")
 	}
 
-	if c.conf.Sleep == "" {
+	log.Print("Pinging server")
+	c.enroll()
+	c.makeKnownHosts()
+
```

Comment:
We should still early exit if c.conf.Sleep == ""

This will happen since we'll fatalf when failing to parse the duration, but with an error